projects/project-player-library/src/lib/pages/details-page/details-page.component.ts

```diff
@@ -201,6 +206,11 @@ export class DetailsPageComponent implements OnInit {
         options.push(actions.CERTIFICATE_ACTION)
       }
     }
+    else {
```

Comment:
Why are we doing this in the else block?
Check how SYNC icon is handled

Reply:
done